corehq/apps/hqwebapp/management/commands/synch_phone_nums.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.models import CommCareUser
+from corehq.apps.sms.tasks import sync_user_phone_numbers as sms_sync_user_phone_numbers
+from corehq.form_processor.models import CommCareCaseSQL
+from corehq.messaging.tasks import sync_case_for_messaging
+from corehq.util.log import with_progress_bar
+from corehq.sql_db.util import get_db_aliases_for_partitioned_query
+
+
+class Command(BaseCommand):
+    help = "Resync all contacts' phone numbers for given projects"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-p', '--projects', type=str, help='Comma separated project list')
```

Comment:
I would change this to 
```py
'-d', '--domains', type=str, help='Comma separated list of domains'
```
since "domain" is the conventional name used throughout HQ.

#### Context: recent slack discussion:

Daniel Miller [12:10 PM]
"domain" vs "project": which word should be used when creating a new management command argument? that is, an argument that accepts the domain name. (edited)

Jonathan Emord [12:16 PM]
ethan and i were just talking about this last week
from that convo:
```
me: 1) Domain is the model name
2) project is often used as an instance of a domain object. project is also the product facing name
3) domain_obj is sometimes used as an instance of a domain object
4) domain is often used as the name of a project (`project.name`)
and non of those are always consistent :slightly_smiling_face:
esoergel [14:38]
My understanding is that in code:
• instances of Domain are called either `project` or `domain_obj` - my preference is the latter
• the string name should be referred to as `domain`
• calling the instance `domain` or calling the string `project` is "wrong" and should be corrected, though I have seen both of those
```

I think that `domain` is the correct answer though

Jenny Schweers [12:17 PM]
agree `domain` is the correct answer